project2/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <div>
+        <h1> Shrimp & Grits </h1>
+        <h2> 30 Minute Recipe </h2>
+        <p>
+            <ol>
+                3/4th Teaspoon Salt
+            </ol>
+            <ol>
+                1/2 Teaspoon Pepper
+            </ol>
+            <ol>
+                2% milk
+            </ol>
+            <ol>
+                1/3 cup butter, cubed
+            </ol>
+
+            <button class="button-reset my-button-style" type="button">Read More
+            </button>
+        </p>
```

Comment:
1. You should  remove the `p` tag on line 17 and 33 -- it's not necessary for wrapping around these other elements. You could later wrap a `div` around them if your layout requires it, but I'd get the basics sorted first.
1. The markup for an ordered list is :
```html
<ol>
  <li>List item</li>
  <li>List item</li>
  <li>List item</li>
</ol>
```
Also, consider if this list is an *ordered* list (numbered) or *unordered*. While it makes sense that the actual steps of the recipe are ordered (`ol`), I don't know if you can make the same case for the ingredients.